packages/deployment/upgrade-test/Dockerfile

```diff
@@ -147,5 +147,6 @@ COPY ./${THIS_NAME} ./upgrade-test-scripts/${THIS_NAME}/
 RUN apt install -y tmux
 SHELL ["/bin/bash", "-c"]
 RUN chmod +x ./upgrade-test-scripts/*.sh
+RUN . ./upgrade-test-scripts/start_to_to.sh
 # enter image in interactive shell
 ENTRYPOINT /usr/src/agoric-sdk/upgrade-test-scripts/start_to_to.sh
```

Comment:
Should we actually close the "target" layer and have a layer just for the interactive shell?

```suggestion
RUN chmod +x ./upgrade-test-scripts/*.sh
SHELL ["/bin/bash", "-c"]
RUN . ./upgrade-test-scripts/start_to_to.sh

# enter image in interactive shell
FROM agoric-upgrade-12
RUN apt install -y tmux
SHELL ["/bin/bash", "-c"]
ENTRYPOINT /usr/src/agoric-sdk/upgrade-test-scripts/start_to_to.sh
```

Reply:
Of course we'd need to have the corresponding target change in the Makefile

Reply:
Ah. Just realized we don't need the tmux install -- the `env_setup` script will install it if it detects an interactive shell.

Aside from that, what's the benefit of separating a layer for the interactive shell?

Reply:
Easier to reason about, and thus less error prone when adding layers. An upgrade layer is always an image with the tests that have run.

Reply:
I think I'd need to understand the layer concerns more. Ignoring the tmux install (which I remove in a following commit as it's redundant), there is no difference between the build-time `agoric-upgrade-11` and an interactive shell one yet (except a different entrypoint which can be changed at runtime)...

Reply:
Actually with the removed tmux, there is this entrypoint question. Should we have an `ENTRYPOINT` for every layer so that one can easily start an interactive container from every layer without having to specify the entrypoint on the command line? Or on the opposite side, should we remove the entrypoint from Dockerfile completely?

Reply:
If we don't do either, I would ask for a new line before the `# enter image in interactive shell`, to visually separate the conceptual "upgrade layer" a little more